cog/core/downtime.py

```
@@ -0,0 +1,97 @@
+        start = datetime.strptime(start_str, DATETIME_FORMAT)
+        if end_str:
+            end = datetime.strptime(end_str, DATETIME_FORMAT)
+        else:
+            end = datetime.now()
+        return Downtime(start, end, is_restored)
+
+    @staticmethod
+    def from_dict(d: dict) -> Downtime:
+        return Downtime.from_str(
+            start_str=d["start"],
+            end_str=d.get("end", None),
```

Ruff (0.8.2)

Use `d.get("end")` instead of `d.get("end", None)`

Replace `d.get("end", None)` with `d.get("end")`

(SIM910)